spec/forms/contributor_form_spec.rb

```diff
@@ -94,11 +94,25 @@
     end
 
     context 'when depositing and missing name' do
+      let(:first_name) { 'Jane' }
+      let(:last_name) { '' }
+
+      it 'is not valid' do
+        expect(form.valid?(:deposit)).to be false
+        expect(form.errors.size).to eq(1)
+        expect(form.errors[:last_name]).to eq(['must provide a last name'])
+      end
+    end
+
+    context 'when depositing and missing part of a name' do
```

Comment:
The context here looks like it's missing the entire name. Maybe the context strings on lines 107 and 96 should be swapped? Not blocking, just wondering.

Reply:
Good catch. Fixing.